### PR TITLE
Initialize CO2 to reasonable value instead of zero in fullchem_mod.F90

### LINKED ISSUE #1529: [BUG/ISSUE] (minor issue) Initialize CO2 to a relevant value in fullchem_mod.F90
Hello all.

We've been looking at the process of error estimation in KPP's Rosenbrock solver, and noticed that in calculating the error norm of a FullChem run, the relative error of CO2 is large. There are several points this raises:
1) The error norm determines the internal substep size. 
2) CO2 is not a stiff or active species. It is essentially for mass balance of carbon oxidation. It is nowhere a reactant.
3) The reason the relative error for CO2 is high is because it is initialized to zero for every timestep. 
4) the error norm should be minimized: non-relevant terms should be excluded or their impacts kept small.

This issue can be resolved by simply setting CO2 to a relevant (or even arbitrarily) large value. Pardon me if I don't propose a number here, because C(ind_CO2) is in molec/cm3, and would be pressure dependent if it were calculated appropriately.

In fullchem_mod.F90, where CO2 is set to zero, there is a 2016 comment from @msulprizio 
```
       ! Temporary fix for CO2                                                                                                                                                                                                                
       ! CO2 is a dead species and needs to be set to zero to                                                                                                                                                                                 
       ! match the old SMVGEAR code (mps, 6/14/16)                                                                                                                                                                                            
       IF ( TRIM( SpcInfo%Name ) == 'CO2' ) THEN
          State_Chm%Species(N)%Conc(:,:,:) = 0.0_fp
       ENDIF
```
... this was right after completing the 'flexchem' upgrade. I suspect we no longer need to worry about SMVGEAR behavior.

tagging @christophkeller 

This is also related to ongoing [KPP work:](https://github.com/KineticPreProcessor/KPP/issues/66)

### PR DESCRIPTION
**Pull request by msulprizio**
As discussed in https://github.com/geoschem/geos-chem/issues/1529, the code to reset CO2 to zero at every timestep should not be used. Instead, @msl3v recommends setting CO2 to an appropriate ambient value (400 ppm).

### CODE REVIEW

**Review by yantosca — CHANGES_REQUESTED** [review]
OK to merge after units are confirmed

File: GeosCore/fullchem_mod.F90 [hunk 1]
```diff
@@ -343,11 +338,9 @@ SUBROUTINE Do_FullChem( Input_Opt,  State_Chm, State_Diag,                 &
           END SELECT
        ENDIF
 
-       ! Temporary fix for CO2
-       ! CO2 is a dead species and needs to be set to zero to
-       ! match the old SMVGEAR code (mps, 6/14/16)
+       ! Set CO2 to 400 ppm (msl, mps, 1/17/23)
        IF ( TRIM( SpcInfo%Name ) == 'CO2' ) THEN
-          State_Chm%Species(N)%Conc(:,:,:) = 0.0_fp
+          State_Chm%Species(N)%Conc(:,:,:) = 400.0e-6_fp ! v/v
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply GeosCore/fullchem_mod.F90:343]: Yes, they are indeed in `kg`. Thanks for catching that. I will push a fix.

[the remark this answers is not in the document]
Comment (msulprizio) [reply GeosCore/fullchem_mod.F90:343]: I tried converting 400 ppb to kg within the code but got KPP errors.

**Comment by msulprizio** [comment]
A better fix is needed here. I will close out this PR for now.

**Pull request closed without merging**